server/templates/includes/faq_waco.html

```diff
@@ -0,0 +1,196 @@
+          and mail it to:
+        </p>
+        <p class="t-serif has-b-btm-marg">
+          <span class="t-sans t-size-s">
+            <em>
+              Waco Local News Initiative<br />
+              The Texas Tribune<br />
+              919 Congress Avenue, Sixth Floor<br />
+              Austin, Texas 78701<br />
+            </em>
+          </span>
+        </p>
+      </details>
+    </li>
+    <li class="c-faq__item">
+      <details>
+        <summary class="c-faq__summary l-pos-rel gtm-custom-qa-dropdown" data-qa-title="FAQ" data-qa-question="What if I want to give more?">
+          <h3 class="t-size-s">For philanthropic support, please reach out to</h3>
+          <span class="c-icon c-icon--baseline t-size-xs c-faq__icon c-icon l-height-full l-pos-abs">
+            <svg aria-hidden="true">
+              <use href="#caret-down"></use>
+            </svg>
+          </span>
+        </summary>
+        <p class="t-serif has-b-btm-marg">
+          Senior Major Gift Officer James Canup at
+            <a href="mailto:james.canup@texastribune.org">james.canup@texastribune.org</a> or (512) 658-4516.
+        </p>
+        <p class="t-serif has-b-btm-marg">
+          To make a tax-deductible gift in the name of a business, contact April Hinkle to join our
+          <a href="/business">Business Membership</a> program.
+        </p>
+        <p class="t-serif has-b-btm-marg">
+          For
+          <a href="https://www.texastribune.org/support-us/corporate-sponsors/">corporate sponsorships</a>
+          and business membership, contact:<br />
+          <span class="t-sans t-size-s">
+            <em>
+              <strong>April Hinkle</strong><br />
+              Chief Revenue Officer<br />
+              512-716-8634<br />
+              <a href="mailto:ahinkle@texastribune.org">ahinkle@texastribune.org</a>
+            </em>
+          </span>
+        </p>
+      </details>
+    </li>
+    <li class="c-faq__item">
+      <details>
+        <summary class="c-faq__summary l-pos-rel gtm-custom-qa-dropdown" data-qa-title="FAQ" data-qa-question="Still have questions?">
+          <h3 class="t-size-s">What if I still have questions?</h3>
+          <span class="c-icon c-icon--baseline t-size-xs c-faq__icon c-icon l-height-full l-pos-abs">
+            <svg aria-hidden="true">
+              <use href="#caret-down"></use>
+            </svg>
+          </span>
+        </summary>
+        <p class="t-serif has-b-btm-marg">
+          For questions about membership:.<br />
+          Leave us a message at 512-993-0166<br />
+          Email us at
+          <a href="mailto:membership@texastribune.org">membership@texastribune.org</a>
+        </p>
+      </details>
+    </li>
+  </ul>
+  <div class="c-faq__login has-padding t-align-right">
+    Already a member? <a class="t-weight-bold has-text-teal" href="/account">Log in</a>
+  </div>
```

Comment:
**suggestion (non-blocking):** I say we remove this call to action to login. I think it's already kinda confusing because this isn't necessarily a membership in the way we're used to. I don't want people to think that when they donate here, they see it in the portal

Reply:
done